### PR TITLE
Rating and Toast Component

### PR DESCRIPTION
**Pull request by Nagashree01**
Added Rating and Toast Component
ratingAndToast

### CODE REVIEW

File: components/rating.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+@import url('/stylesheets/base.css');
+
+.select {
+    color: gold;
```
Comment (munjalSaksham71) [inline components/rating.css:4]: Can you use css variable here?

**Review by munjalSaksham71 — APPROVED** [review]
Looks Good, Just added a minor comment. And then you can mege it.

File: stylesheets/base.css [hunk 2]
```diff
@@ -33,6 +34,11 @@
     --danger-color:#ff7875;
     --primary-alert-color:#69c0ff;
     --secondary-alert-color:#eaeaec;
+    /* rating color */
+    --gold-color:gold;
```
Comment (munjalSaksham71) [inline stylesheets/base.css:38]: for simplifications,  use #FFD700. the hex-code for gold.

Comment (Nagashree01) [reply stylesheets/base.css]: yes will do this

**Review by SuyashTiwariCoder — APPROVED** [review]
Looks good you can merge

**Pull request merged**